Week-01/guessing-game.js

```diff
@@ -42,19 +42,19 @@ dinosaur = 8;
 dinosaur = 999999;
 
 // Q: In the code above, what is the value of `dinosaur`?
-// A:
+// A: -200
 // Q: In the code above, all of the variables above are "number" type. What do you think are the possible values of the number type?
-// A:
+// A: any kind of values
```

Comment:
Only numbers are possible values of the number type. Numbers can be whole integers (1,2,5, 10) or numbers with decimals (4.5, 0.004, 21.99999) and negative numbers (-1, -2, -5, -10).